frappe/desk/page/setup_wizard/setup_wizard.js

```diff
@@ -529,16 +533,19 @@ frappe.setup.utils = {
 
 		var country_field = slide.get_field('country');
 
-		slide.get_input("country").empty()
-			.add_options([""].concat(Object.keys(data.country_info).sort()));
 
-		slide.get_input("currency").empty()
-			.add_options(frappe.utils.unique([""].concat(
-				$.map(data.country_info, opts => opts.currency)
-			)).sort());
+		country_field.set_data([__("Select Country")].concat(Object.keys(data.country_info).sort()));
+
+		slide.get_input("currency")
+			.empty()
+			.add_options(
+				frappe.utils.unique(
+					["Select Currency"].concat($.map(data.country_info, opts => opts.currency).sort())
```

Comment:
Why is "Select Country/Timezone/Currency" set as an option if we are adding a placeholder?

Reply:
Hmm, they were default options before, but I think the county field doesn't need one now because it's a autocomplete field. A placeholder will be enough. 
TImezone and currency fields have default selected options as soon as  on_load() runs. But the placeholder is on-screen till then.